### PR TITLE
Rails React Combo

### PR DESCRIPTION
**Pull request by jonas-45**
In this PR, I have done the following. 👇

- [ ] Created a new Rails app called 'greetings'.
- [ ] Used the command rails new -j webpack my_new_rails_app as described in the previous lesson's tutorial.
- [ ] Verified that the jsbundling-rails gem is added to your Gemfile.
- [ ] Initialized project with Git.
- [ ] Installed needed node dependencies as described in the previous lesson's tutorial.
- [ ] Created a babel.config.js in the root folder of your project as described in the previous lesson's tutorial.
- [ ] Updated webpack.config.js to include the babel loader as described in the previous lesson's tutorial.
- [ ] Made sure that your project has a Postgres database set up.
- [ ] Created a table for storing your messages and create 5 different greetings.
- [ ] Created an API endpoint that selects a random greeting from table
- [ ] Created a static view that will be the root of your app.
- [ ] Created the App component with react-router and render it in your static view.
- [ ] Created the Greeting component that will display a greeting. Set it up as a route in your App component.
- [ ] Created a store, an action, and a reducer that will connect to your API endpoint to get the random greeting.
- [ ] Displayed the random greeting in your Greeting component.
- [ ] Fix linter errors

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  gem 'debug', platforms: %i[mri mingw x64_mingw]
+end
+
+group :development do
+  # Use console on exceptions pages [https://github.com/rails/web-console]
+  gem 'web-console'
+
+  # Add speed badges [https://github.com/MiniProfiler/rack-mini-profiler]
+  # gem "rack-mini-profiler"
+
```
Comment (rloterh) [inline Gemfile:65]: - [ ] Random greetings not generating as intended due to CORS configuration not set correctly. When the generate greeting button is clicked, CORS Policy error is thrown in the browser console, and no greetings generated. Let's improve upon this by adding necessary CORS configuration/ gem and ensure that random greetings are generating correctly.

image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Nuta, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 
🏁  🌴 🚦 🚗 🏫  🏡   _🏃 

## Highlights 👍  
- Good job so far connecting backend and feontend in one app!

## Required Changes ♻️
_Kindly check the inline comments under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @jonas-45 , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Awesome joj implementing the requested change. 🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jonas-45** [comment]
Thanks @ridwanediallo for your time and effort in going through my work

**Pull request merged**